src/server.cpp

```diff
@@ -1084,6 +1085,7 @@ void Server::AsyncRunStep(float dtime, bool initial_step)
 	m_shutdown_state.tick(dtime, this);
 }
 
+//! Receive all incoming packets and wait for at least `timeout` seconds.
 void Server::Receive(float timeout)
```

Comment:
We usually only use doxygen in headers. https://dev.minetest.net/Code_style_guidelines#Comments